examples/cndpfwd/main.c

```diff
@@ -194,14 +194,29 @@ _txonly_test(jcfg_lport_t *lport, struct fwd_info *fwd)
             pktmbuf_t *xb = tx_mbufs[j];
             uint64_t *p   = pktmbuf_mtod(xb, uint64_t *);
 
-            p[0]                 = 0xfd3c78299efefd3c;
-            p[1]                 = 0x00450008b82c9efe;
-            p[2]                 = 0x110400004f122e00;
-            p[3]                 = 0xa8c00100a8c01e22;
-            p[4]                 = 0x1a002e16d2040101;
-            p[5]                 = 0x706f6e6d6c6b9a9e;
-            p[6]                 = 0x7877767574737271;
-            p[7]                 = 0x31307a79;
+            /*
+             * IPv4/UDP 64 byte packet
+             * TTL/Port Src/Dest   :       64/ 1234/ 5678
+             * Pkt Type:VLAN ID    :      IPv4 / UDP:0001
+             * 802.1p CoS/DSCP/IPP :            0/  0/  0
```

Comment:
Minor nit: why do you mention VLAN and VxLAN in the comments here? This is just a normal IP/UDP packet, right?

Reply:
I created this packet with pktgen and then I just grabbed the screen text for the packet. I can remove it if you like.